README.md

```diff
@@ -29,17 +29,32 @@ marks](CaliptraTrademarkPolicy.md).
 * [FMC](https://github.com/chipsalliance/caliptra-sw/tree/main/fmc#readme)
 * [Firmware](https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md)
 
+## Security policy
+
+For information on the Caliptra security policy, how to report a vulerability,
+and published security advisories, refer to
+[Caliptra security policy](https://github.com/chipsalliance/caliptra/security/policy).
```

Comment:
Is this link correct? Shouldn't it point to h://github/chips/caliptra/Security.md ?

Reply:
This would be the preferred location. The link won't look right until we have the security policy enabled. This is a bit of a chicken and egg problem when setting this up. I experimented with a dummy repository to see how everything would work. 

Here is my dummy repo to see what it would look like. SECURITY.md gets displayed on the page: https://github.com/FerralCoder/Test/security/policy